tests/proxy_test.go

```diff
@@ -218,10 +218,11 @@ type proxy struct {
 }
 
 func runGRPCProxyServer() (proxy, func(), error) {
-	return runGRPCProxyServerWithServerCount(1)
+	p, _, cleanup, err := runGRPCProxyServerWithServerCount(1)
+	return p, cleanup, err
```

Comment:
Can we make the parameters returned consistent even though it'll probably be another _ assignment?

Reply:
I was trying to reduce the ripple caused by the signature change. Otherwise we would simply use runGRPCProxyServerWithServerCount every where :)